gh_actions_exporter/metrics.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, settings: Settings):
         self.job_labelnames = self.common_labelnames.copy() + [
             "job_name",
             "runner_type",
+            "runner_labels",
```

Comment:
What kind of query or filter were you trying to do when deciding to add the `runner_labels`  and have you tried using the [relabel option](https://github.com/scality/gh-actions-exporter/blob/55727247aac272072ad1ccf2197aeeb7723d879c/gh_actions_exporter/config.py#L38)?

I see no issue in adding it, just wanna make sure you tried all the options available to you.

For example in our org, our self-hosted runners are defined with multiple labels:
- The os (`ubuntu-20.04`)
- The flavor (`large`)
- And in some case the cloud provider (`aws`, `azure`, etc)

And we use the relabel feature to retrieve those value and create new label like:
- `cloud=aws`
- `flavor=large`

Which makes it easier for us to query around.

Reply:
This does look promising. Would you have an example config for achieving the example above?

Reply:
I do yes, this is an example of what we've been playing with:

```yaml
job_relabelling:
  - label: cloud
    default: github-hosted
    values:
      - azure
      - aws
    type: name
  - label: image
    default: unknown
    values:
      - ubuntu-latest
      - ubuntu-20.04
      - ubuntu-22.04
      - rocky-8
      - redhat-8
      - redhat-9
  - label: flavor
    default: medium # consider github-hosted flavors as medium
    values:
      - small
      - medium
      - large
      - xlarge
```

And sorry for the lack of documentation on those 😅 , I'll take note on my end and make an effort towards that in the coming days.